### PR TITLE
add docker files

### PR DESCRIPTION
**Pull request by akhilambati**
add docker files

### CODE REVIEW

File: run_zeserver.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cp .env-local .evn
```
Comment (abskrj) [inline run_zeserver.sh]: evn will be env?

Comment (akhilambati) [reply run_zeserver.sh]: sorry wrong typo, corrected it

Comment (akhilambati) [reply run_zeserver.sh]: @abhishekraj272 if you need any enhancements or suggestions , you can specify..

Comment (abskrj) [reply run_zeserver.sh]: @akhilambati sure

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+RUN apt install -y ffmpeg
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /build
+
+#COPY go.mod .
```
Comment (abskrj) [inline Dockerfile:15]: why these are comments?

Comment (akhilambati) [reply Dockerfile:15]: i can remove that, basically this commented code was initially used for creating and testing  the image for zestream server

**Pull request closed without merging**